modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -49,30 +50,32 @@ namespace {{packageName}}.Client
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" /> class
-        /// with default configuration and base path ({{{basePath}}}).
+        /// with default configuration and base path (<see cref="SERVER_DEFAULT_ADDRESS"/>).
         /// </summary>
-        public ApiClient()
-        {
+        public ApiClient() {
+            log.Trace("start");
             Configuration = Configuration.Default;
-            RestClient = new RestClient("{{{basePath}}}");
+            RestClient = new RestClient(SERVER_DEFAULT_ADDRESS);
             {{#netStandard}}
             RestClient.IgnoreResponseStatusCode = true;
             {{/netStandard}}
+            log.Trace("end");
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" /> class
-        /// with default base path ({{{basePath}}}).
+        /// with default base path (<see cref="SERVER_DEFAULT_ADDRESS"/>).
```

Comment:
`SERVER_DEFAULT_ADDRESS` is private, so there's nothing to *see* if the client generates docs.